monitoring/prometheus/prometheus.go

```diff
@@ -0,0 +1,115 @@
+
+package prometheus
+
+import (
+	prom "github.com/prometheus/client_golang/prometheus"
```

Comment:
Nit: I would group the imports by …

1. standard library
2. first-party
3. third-party